app/src/main/java/com/burhanrashid52/photoeditor/EditImageActivity.java

```diff
@@ -174,9 +174,24 @@ public void onEditTextChangeListener(final View rootView, String text, int color
                 TextEditorDialogFragment.show(this, text, colorCode);
         textEditorDialogFragment.setOnTextEditorListener(new TextEditorDialogFragment.TextEditor() {
             @Override
-            public void onDone(String inputText, int colorCode) {
+            public void onDone(String inputText, int colorCode , boolean bold , boolean italic) {
```

Comment:
Passing multiple parameters in a function is not the best practice. For styling the text please check `TextStyleBuilder` and add these parameters to that class.